specification/signalr/resource-manager/readme.md

```diff
@@ -60,6 +60,15 @@ directive:
     where:
     - $.definitions.Sku
     reason: It's required by spec of enumerating SKUs for an existing resource
+  - suppress: ParametersOrder
+    where:
+    - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/signalR/{resourceName}/privateEndpointConnections/{privateEndpointConnectionName}"].get
+    - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/signalR/{resourceName}/privateEndpointConnections/{privateEndpointConnectionName}"].put
+    - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/signalR/{resourceName}/privateEndpointConnections/{privateEndpointConnectionName}"].delete
+    - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/signalR/{resourceName}/sharedPrivateLinkResources/{sharedPrivateLinkResourceName}"].get
+    - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/signalR/{resourceName}/sharedPrivateLinkResources/{sharedPrivateLinkResourceName}"].put
+    - $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/signalR/{resourceName}/sharedPrivateLinkResources/{sharedPrivateLinkResourceName}"].delete
+    reason: It can introduce a breaking change when updating parameter order, since SignalR service has already shipped public versions.
```

Comment:
Hi @juniwang , have you released SDKs? if not, suggest to fix the error, as it only affects parameter order in SDK methods.

Reply:
Yes we have released the SDKs.

Reply:
Given that our SDKs were released, I think it's better to keep original order to avoid breaking changes. @leni-msft  any other comments/suggestions?

Reply:
ok, sounds good to me.